integration_tests/pages/overview/EducationAndTrainingPage.ts

```diff
@@ -40,13 +40,18 @@ export default class EducationAndTrainingPage extends Page {
     return this
   }
 
-  hasCompletedInPrisonQualificationsDisplayed(): EducationAndTrainingPage {
-    this.completedInPrisonQualificationsTable().should('be.visible')
+  hasCompletedInPrisonQualificationsLast12MonthsDisplayed(): EducationAndTrainingPage {
+    this.completedInPrisonQualificationsLast12MonthsTable().should('be.visible')
```

Comment:
So this is just asserting the table is visible .. 🤔
I wonder if that is part of the problem in this PR .... to be able to assert there are completed courses displayed, we need to assert the table has some records (<tr>s perhaps?), and/or does also contain the "no records" message

Reply:
I think because the designs have the empty state message "No courses or qualifications completed in last 12 months" as an actual table row; make it a little more difficult to test this. 

Rather than having a Table for data, then a paragraph element for the empty state message (which probably would also be more semantically correct) - which would clear up the confusion here I think